en/04_Changelogs/5.0.0.md

```diff
@@ -62,6 +63,29 @@ This is a major release and as a result there are a number of breaking API chang
     - `SilverStripe\ORM\Connect\Query`
   - Removed `SilverStripe\ORM\DataList::getGenerator()` (use `getIterator()` instead)
   - Removed the `SilverStripe\ORM\Map_Iterator` class. `SilverStripe\ORM\Map` now uses a generator instead.
+- Removed various API related to `SwiftMailer`
+  - Removed the following classes and interfaces
+    - `SilverStripe\Control\Email\Mailer`
+    - `SwiftMailer`
+    - `SwiftPlugin`
```

Comment:
If these had namespaces, please put the FQCN here.

Reply:
Updated